crates/wascc-logging/src/lib.rs

```diff
@@ -0,0 +1,150 @@
+// Copyright 2015-2019 Capital One Services, LLC
```

Comment:
We should probably note here that this is adopted from the wascc logging crate with the original license included here